index.html

```diff
@@ -239,6 +239,8 @@ <h2 class="success__title">Изображение успешно загруже
     </li>
   </template>
 
+  <script src = "./vendor/nouislider/nouislider.js"></script>
```

Comment:
```suggestion
  <script src="vendor/nouislider/nouislider.js"></script>
```

Пробелы тут не нужны - это html файл))